slack-api-client/src/main/java/com/slack/api/SlackConfig.java

```diff
@@ -213,9 +221,30 @@ private static String initProxyUrl() {
 
     private String legacyStatusEndpointUrlPrefix = LegacyStatusClient.ENDPOINT_URL_PREFIX;
 
-    private MethodsConfig methodsConfig = MethodsConfig.DEFAULT_SINGLETON;
+    @Builder.Default
+    private ExecutorServiceProvider executorServiceProvider = DaemonThreadExecutorServiceProvider.getInstance();
+
+    private MethodsConfig methodsConfig = new MethodsConfig();
```

Comment:
Changed these config properties from immutable to mutable. Some people may prefer immutable objects here but I don't think that this is a breaking change.